django_netjsonconfig/api/generics.py

```diff
@@ -10,7 +24,7 @@ class BaseSearchTemplate(ListAPIView):
     def get_queryset(self):
         name = self.request.GET.get('name', None)
         des = self.request.GET.get('des', None)
-        temp = self.template_model.filter(sharing='public')
+        temp = self.template_model.objects.filter(sharing='public')
```

Comment:
what is `des` bytheway?

Reply:
`/search/` is used to list all public templates as well as search for templates by `name` and `description`. it can be used as follows:
`search/`= list all public templates
`search/?name=n`= list all public templates with name equal to `n` or contains `n`
`search/?des=d` = list all public templates with description equal to `d` or contains `d`
`search/?name=n&&des=d` = list all public templates with name equal to `n` or contains `n` and description equal to `d` or contains `d`.